packages/cadl-ranch-specs/http/parameters/spread/main.tsp

```diff
@@ -224,22 +260,27 @@ namespace Alias {
     @header
     `x-ms-test-header`: string;
 
-    prop1: string;
-    prop2: string;
-    prop3: string;
-    prop4: string;
-    prop5: string;
-    prop6: string;
+    /** required string */
+    requiredString: string;
+
+    /** optional int */
+    optionalInt?: int32;
+
+    /** required int */
+    requiredIntList: int32[];
+
+    /** optional string */
+    optionalStringList?: string[];
   };
 
   @scenario
   @scenarioDoc("""
   Test case for spread alias including 6 parameters. May handle as property bag for these parameters.
   
-  Should not generate any model named `AliasMultipleRequestParameters`.
+  Should not generate any model named `MultipleRequestParameters`.
   Should generate an operation like below:
```

Comment:
Since it contains both optional properties and required properties, the method signature might vary across different languages.